docs/data-modeling/field-types.md

```diff
@@ -45,23 +47,33 @@ Metabase recognizes the following field types:
 - **Income**
 - **Price**
 
-### Numeric
+See [Currency formatting options](../questions/sharing/visualizations/table.md#currency-formatting-options).
+
+## Numeric
+
+Metabase will treat numeric field types as [numbers](../questions/sharing/visualizations/table.md#number-formatting-options).
 
-- **Percentage**
-- **Quantity**
-- **Score**
-- **Share**. The same as percentage, so prefer "Percentage".
+- **Percentage**: Displays the number as a percentage by default.
+- **Quantity**: Displays the number as normal by default.
+- **Score**: Displays the number as normal by default.
+- **Share**: The same as percentage, so prefer "Percentage".
 
-### Profile
+See [Number formatting options](../questions/sharing/visualizations/table.md#number-formatting-options).
 
-- **Birthday**
-- **Company**
-- **Email**
-- **Owner**
-- **Subscription**
-- **User**
+## Profile
 
-### Date and Time
+Fields that deal with people.
+
+- **Birthday**: Everyone has a birthday, but birthdays only happen once a year.
+- **Company**: Text field.
+- **Email**: Displays as a [mailto](https://en.wikipedia.org/wiki/Mailto) link.
+- **Owner**: Text field.
```

Comment:
Same note as on Company: is there anything special that Metabase does with fields set to "Owner", "Subscription", etc?